### PR TITLE
Install Firefox

### PR DESCRIPTION
**Pull request by pjmelling**
Install Firefox for feature testing.

### CODE REVIEW

**Comment by pjmelling** [comment]
@benhutton This installs Firefox. I left Chrome in place to make the transition. We would merge this first, then merge the switch to the FF driver in the www project. After that, we could remove Chrome to clean up the the Dockerfile. How does that sound?

File: base/Dockerfile [hunk 1]
```diff
@@ -9,23 +9,33 @@ RUN apt-get install -y \
       redis-tools \
       libnss3-tools
 
+RUN FIREFOX_URL='https://ftp.mozilla.org/pub/firefox/releases/61.0.2/linux-x86_64/en-US/firefox-61.0.2.tar.bz2' \
```
Comment (benhutton) [inline base/Dockerfile]: @pjmelling I feel like hardcoding the version here is setting us up for more work than we need. Right now, chrome is automatically installing the latest every time the image is compiled. I'd think that until this bites us, we should do the same for Firefox. Do you agree? Looks like https://ftp.mozilla.org/pub/firefox/releases/latest/README.txt gives instructions on how to do that....

Comment (pjmelling) [reply base/Dockerfile]: Yes, I agree. Picking latest is a much better way. I'll make that change right now.

File: base/Dockerfile [hunk 2]
```diff
@@ -96,24 +88,6 @@ RUN apt-get install -y --no-install-recommends \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/*
 
-RUN set -x  \
```
Comment (benhutton) [inline base/Dockerfile]: @pjmelling I think `phantomjs` is for `teaspoon`

Comment (pjmelling) [reply base/Dockerfile]: @benhutton Ah yes, you're right. I was wondering what would break!

**Comment by pjmelling** [comment]
@benhutton I've got this into shape now.

**Pull request merged**